composer/utils/fx_utils.py

```diff
@@ -111,6 +113,40 @@ def replace_op(gm: GraphModule, src_ops: Union[Callable, str, List[Union[Callabl
     return gm
 
 
+class BlockStochasticModule(nn.Module):
```

Comment:
should this (and the corresponding code) be in the `stochastic_depth` algorithm instead? prefer to keep all algorithms as standalone as possible.

Reply:
Sure. Will move it there.

Reply:
I expect the functionality here to replace the current stochastic depth implementation. Should that happen in this PR or a future PR?

Reply:
This will happen in a followup PR.

Reply:
Moved to stochastic_layers.py